src/libraries/System.Security.Cryptography/tests/X509Certificates/CertTests.cs

```diff
@@ -571,6 +571,86 @@ public static void SignatureAlgorithmOidReadableForGostCertificate()
             }
         }
 
+        [ConditionalFact(typeof(PlatformSupport), nameof(PlatformSupport.PlatformCryptoProviderFunctional))]
+        [OuterLoop("Hardware backed key generation takes several seconds.")]
+        public static void CreateCertificate_MicrosoftPlatformCryptoProvider_EcdsaKey()
+        {
+             CngKey key = null;
+
+            try
+            {
+                CngKeyCreationParameters cngCreationParameters = new CngKeyCreationParameters
+                {
+                    Provider = CngProvider.MicrosoftPlatformCryptoProvider,
+                    KeyCreationOptions = CngKeyCreationOptions.OverwriteExistingKey,
+                };
+
+                key = CngKey.Create(
+                    CngAlgorithm.ECDsaP384,
+                    nameof(CreateCertificate_MicrosoftPlatformCryptoProvider_EcdsaKey),
+                    cngCreationParameters);
+
+                using (ECDsaCng ecdsa = new ECDsaCng(key))
+                {
+                    CertificateRequest req = new CertificateRequest("CN=potato", ecdsa, HashAlgorithmName.SHA256);
+
+                    using (X509Certificate2 cert = req.CreateSelfSigned(DateTimeOffset.UtcNow, DateTimeOffset.UtcNow))
+                    using (ECDsa certKey = cert.GetECDsaPrivateKey())
+                    {
+                        Assert.NotNull(certKey);
+                        byte[] data = new byte[] { 12, 11, 02, 08, 25, 14, 11, 18, 16 };
+                        byte[] signature = certKey.SignData(data, HashAlgorithmName.SHA256);
+                        bool valid = ecdsa.VerifyData(data, signature, HashAlgorithmName.SHA256);
+                        Assert.True(valid, "valid signature");
```

Comment:
subjective opinion: we can simplify the check. The test contains a single `Assert.True` so extra information seems redundant to me, but I know it's a matter of personal preference or existing convention so you can just ignore this comment.

```suggestion
                        Assert.True(ecdsa.VerifyData(data, signature, HashAlgorithmName.SHA256));
```

Reply:
One nit I get from Jeremy often is line length. Generally we try to keep them under 120 chars. When a line gets too long, I start splitting things out in the variables. I could start wrapping things like:

```
Assert.True(
   ecdsa.VerifyData(data, signature, HashAlgorithmName.SHA256),
   "valid signature");
```

But I dislike that more than creating variables.

The second nit I usually get is that `Assert.True` (and `Assert.False`) should always have a string reason. We don't do it consistently, but I think going forward, at least in System.Security, we always try to use them so that unit test output is more helpful.

Reply:
@vcsjones thanks for the explanation, I'll update my reviewing settings for System.Security